### PR TITLE
Update to the last version of atoum

### LINKED ISSUE #82: Update atoum to 2.*
Please update the dependency with atoum to the lastest versions

### PR DESCRIPTION
**Pull request by marmotz**
Try to fix #82

### CODE REVIEW

File: composer.json [hunk 1]
```diff
@@ -22,7 +22,7 @@
         "symfony/finder": "2.*",
         "symfony/dom-crawler": "2.*",
         "symfony/css-selector": "2.*",
-        "atoum/atoum": "~1.0",
+        "atoum/atoum": "~2.0",
```
Comment (jubianchi) [inline composer.json]: `>=1.0,<3.0` ?

Comment (marmotz) [inline composer.json]: `*` ?

Comment (marmotz) [inline composer.json]: No, you're right

**Comment by FlorianLB** [comment]
Why put a max boundary ?

**Comment by marmotz** [comment]
Because you don't if the next major release will be also compatible with this bundle.

**Comment by stephpy** [comment]
:+1:  @marmotz

**Comment by marmotz** [comment]
Ok, so.... ?

**Comment by FlorianLB** [comment]
Thx !

**Pull request merged**